Meshtastic/Views/Onboarding/OnboardingView.swift

```diff
@@ -0,0 +1,315 @@
+import CoreBluetooth
+import OSLog
+import SwiftUI
+import Foundation
+import MapKit
+
+struct OnboardingView: View {
+	enum SetupGuide: Hashable {
+		case notifications
+		case location
+		case mqtt
+	}
+
+	@State var navigationPath: [SetupGuide] = []
+	@EnvironmentObject var bleManager: BLEManager
+
+	@Environment(\.dismiss) var dismiss
+
+	/// The Title View
+	var title: some View {
+		VStack {
+			Text("Welcome to")
```

Comment:
Just a note that `Welcome to Meshtastic` may have different grammar rules when translated into other languages so separating the sentence into two localized blocks has the potential to cause invalid grammar.

Also, [it is best](https://developer.apple.com/videos/play/wwdc2021/10221/?time=447) for all localized strings to have comments in order to provide context to translators. 
- the context of the view where a localized string is used may have impact on how the string should be translated as the context may affect grammar / or [form](https://developer.apple.com/videos/play/wwdc2021/10221/?time=294) of the translation (e.g. femine, masculine) 
- plural rules should be used when dealing with numbers as different languages have different requirements (for example, [Russian requires more cases](https://developer.apple.com/videos/play/wwdc2021/10221/?time=1183) when dealing with plurals).

```suggestion
			Text("Welcome to", 
			      comment: "Title: Welcome message for the onboarding view, will precede the 'Meshtastic' brand (example: 'Welcome to \"Meshtastic\"')")
```